types/magic-number/index.d.ts

```diff
@@ -1 +1,4 @@
-export declare function detectFile(file: string): string;
+/// <reference types="node" />
+
+export function detectFile(file: string): string;
+export function detectType(data: Buffer): string;
```

Comment:
Does it have to be a Buffer? The code doesn't seem to actually require it, and the docs say "any" (which isn't great but to me hints that there's some smaller interface than Buffer that'd work).

Reply:
Done.